pyvisa-py/tcpip.py

```diff
@@ -365,7 +365,7 @@ def read(self, count):
         end_char, _ = self.get_attribute(constants.VI_ATTR_TERMCHAR)
         enabled, _ = self.get_attribute(constants.VI_ATTR_TERMCHAR_EN)
         timeout, _ = self.get_attribute(constants.VI_ATTR_TMO_VALUE)
-        timeout /= 1000
+        timeout = float(timeout)/1000.0
```

Comment:
You could consider adding 
```python
from __future__ import division
```
to the beginning of the file which would make this useless Alternatively dividing by 1000.0 is enough no need to additionally cast to float explicitely.

Reply:
addressed